.github/actions/code-scanning/action.yaml

```diff
@@ -13,6 +13,9 @@ inputs:
   FR_ARTIFACTORY_USER_ENCRYPTED_PASSWORD:
     description: What user password to use to auth to Maven Server
     required: true
+  githubAction:
+    description: Action performed will decide what steps to use
+    required: true
```

Comment:
perhaps actionTrigger/trigger is a better name?

I find `githubAction` confusing as the action we want to perform is a scan of the code.

Reply:
Yeah good point!